README.md

```diff
@@ -136,3 +142,29 @@ mostly works for an idea of the basics, just replace Neography-specific commands
 ## Contributions
 
 Pull requests and maintanence help would be swell. In addition to being fully tested, please ensure rubocop passes by running `rubocop` from the CLI.
+
+### Running Tests:
+
+Upon first running `bundle exec rake spec` or `bundle exec rspec spec`, you may run into an issue like this:
+
+```
+Failure/Error: Restaurant.delete_all
+
+Neo4j::DeprecatedSchemaDefinitionError:
+            Some schema elements were defined by the model (which is no longer supported), but they do not exist in the database.  Run the following to create them:
+
+  rake neo4j:generate_schema_migration[constraint,Restaurant,uuid]
+
+
+  And then run `rake neo4j:migrate`
+...
+```
+
+To resolve, simply run the commands as it says (but maybe prefix `bundle exec` depending on your setup):
```

Comment:
Ah, I see that you have this in the docs (which is great!).  It would be nicer, though, if people didn't need to think about it